bootstrap/values-bootstrap.yaml

```diff
@@ -1,31 +1,3 @@
 bootstrap-project:
   enabled: true
   prefix: "labs"
-    
-### Argo CD is part of bootrap as all the App CRs will require it to be in existance
-
-argo-cd:
-  enabled: true
-  installCRDs: false
```

Comment:
Is there a way we can toggle this on/ off too? Im imagining someone might already have a cluster wide Argo Instance and just want to run the bootrap